recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/system/stubdom-helpers.te

```diff
@@ -119,3 +134,41 @@ allow audio_helper_t self:shm create_shm_perms;
 init_dontaudit_use_fds(audio_helper_t)
 init_dontaudit_use_script_fds(audio_helper_t)
 xc_dontaudit_read_etc_files(audio_helper_t)
+
+########################################
+#
+# Local Policy: varstored/varstored-watch
+#
+
+# varstored-watch executing & monitoring varstored
+allow varstored_t initrc_t:fd use;
+allow varstored_watch_t bin_t:dir search;
+allow varstored_watch_t initrc_t:unix_stream_socket connectto;
+allow varstored_watch_t varstored_exec_t:file execute_no_trans;
```

Comment:
Don't you want a trans here so that varstored runs as varstored_t?

I think you want `varstored_domtrans(varstored_watch_t)` here.